server/src/core/server/services/users/delete.ts

```diff
@@ -290,14 +289,21 @@ async function deleteUserComments(
     isArchived
   );
 
+  const bundle = i18n.getBundle(tenant.locale);
+  const translatedExplanation = translate(
+    bundle,
+    "User account deleted",
```

Comment:
Do we want this fallback text to match the translation text? (so, `User account was deleted.`)?